main.go

```diff
@@ -50,17 +49,16 @@ To learn more about Go package argument, run "go help packages".`
 )
 
 func init() {
-	// Change glog default log level to INFO.
-	// Note glog is not initialized yet, so we can only use fmt for printing
-	// errors.
+	// Change klog default log level to INFO.
+	klog.InitFlags(nil)
```

Comment:
What's the default logging level for klog? If it's info, we can remove the next two statement that override two flags.

Reply:
It's error

https://github.com/kubernetes/klog/blob/02fe3234c86b222bc85406122bd0d11acbe5363a/klog.go#L401-L403